app/assets/stylesheets/base/layout.css.scss

```diff
@@ -27,6 +27,10 @@ html { overflow-y: auto; }
   &:hover { i { color: white; }}
 }
 
+img, iframe {
+  max-width: 100%;
+}
+
```

Comment:
It shouldn't, we're basically say and image or iframe _isn't_ allowed to be larger than it's parent container.